src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/pagination/Pagination.kt

```diff
@@ -1,22 +1,17 @@
 package uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.pagination
```

Comment:
Really the classes in this package should be in the gateway package (or under it), as it is only used there

Reply:
I'll refactor